packages/itwin/tree-widget/api/tree-widget-react.exports.csv

```diff
@@ -7,18 +7,15 @@ beta;HierarchyVisibilityHandler
 beta;IModelContentTreeComponent:
 public;ModelsTreeComponent:
 beta;ModelsTreeVisibilityHandlerOverrides
-beta;ReportUsageCallback
 public;SelectableTree(props: SelectableTreeProps): JSX.Element | null
-beta;Tree({ getSchemaContext, hierarchyLevelSizeLimit, imodelAccess: providedIModelAccess, ...props }: TreeProps_3): JSX.Element
+beta;TelemetryContextProvider({ children, onPerformanceMeasured, onFeatureUsed, componentIdentifier, }: PropsWithChildren
+beta;Tree({ getSchemaContext, hierarchyLevelSizeLimit, selectionStorage, imodelAccess: providedIModelAccess, ...props }: TreeProps_3): JSX.Element
 public;TreeDefinition
```

Comment:
Maybe rename `TreeDefinition` to `SelectableTreeDefinition` to make it clear what component it's related to?